src/components/display/Table.test.tsx

```diff
@@ -86,11 +86,11 @@ describe('Table', () => {
 			.getAllByRole('row')
 			.find((row) => within(row).queryByText('row1col1') !== null) as HTMLElement;
 		expect(row1).toBeDefined();
+		const checkboxes = await screen.findAllByTestId(ICONS.checkboxOff);
 		fireEvent.focus(row1);
-		await screen.findByTestId(ICONS.checkboxOff);
-		expect(screen.getByTestId(ICONS.checkboxOff)).toBeVisible();
+		expect(checkboxes[1]).toBeInTheDocument();
```

Comment:
Unfortunately this assertion is not really useful: the checkboxes are all in the document, and their visibility is managed by the css. Moving this line before the focus event does not make the test fail.
Unfortunately for us part 2, we cannot test that the checkbox becomes visible on css focus or hover. But we can check, even if it is not the best solution, that the styles are set as we expect. I'm working on updating these test accordingly